rts/at/tx/pom.xml

```diff
@@ -38,9 +38,18 @@
     <ssl.args>-Drts.usessl=true -Djavax.net.ssl.trustStore=keystore.jks -Djavax.net.ssl.trustStorePassword=123456</ssl.args>
     <!-- jakarta.servlet-api version 6.0.0 does not include jakarta/servlet/SingleThreadModel needed for undertow-->
     <version.jakarta.servlet.jakarta-servlet-api>5.0.0</version.jakarta.servlet.jakarta-servlet-api>
-
   </properties>
-
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
```

Comment:
These dependency changes impacted on LRA test https://ci-jenkins-csb-narayana.apps.ocp-c1.prod.psi.redhat.com/job/btny-pulls-narayana/PROFILE=LRA,jdk=openJDK17,label=jnlp-agent/82/consoleFull. I will have a look at it.

Reply:
I am sorry, those LRA error are not related to your PR. (see https://github.com/jbosstm/narayana/pull/2114).